nextcloudappstore/core/templates/user/delete-account.html

```diff
@@ -0,0 +1,16 @@
+{% extends "user/base.html" %}
+{% load i18n %}
+{% load account socialaccount css_class %}
+
+{% block head-title %}{% trans "Delete account" %} - {% endblock %}
+
+{% block account-content %}
+    <h1>{% trans "Delete account" %}</h1>
+    <section>
+        <p class="alert alert-danger">{% trans 'Deleting your account will delete all your apps and releases as well. To delete your account, insert your e-mail address into the input field and click the "Delete account" button:' %}</p>
+        <form method="POST" action="{% url 'account-deletion' %}">
+            {% include 'user/form-fields.html' %}
+            <p><input type="submit" class="btn btn-primary btn-block" value="{% trans 'Delete account' %}"></p>
```

Comment:
Just a design suggestion: Instead of `btn-primary` we could maybe use `btn-danger` here, which makes the button red. In that case `.btn-danger` should be added to [this rule in theme.css](https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/static/assets/css/theme.css#L65).